frontera/contrib/backends/remote/codecs/msgpack.py

```diff
@@ -6,31 +6,11 @@
 from msgpack import packb, unpackb
 
 from frontera.core.codec import BaseDecoder, BaseEncoder
-import six
 from w3lib.util import to_native_str
 
 
 def _prepare_request_message(request):
-    def serialize(obj):
-        """Recursively walk object's hierarchy."""
-        if isinstance(obj, six.text_type):
-            return obj.encode('utf8')
-        if isinstance(obj, (bool, six.integer_types, float, six.binary_type)):
-            return obj
-        elif isinstance(obj, dict):
-            obj = obj.copy()
-            for key in obj:
-                obj[key] = serialize(obj[key])
-            return obj
-        elif isinstance(obj, list):
-            return [serialize(item) for item in obj]
-        elif isinstance(obj, tuple):
-            return tuple(serialize([item for item in obj]))
-        elif hasattr(obj, '__dict__'):
-            return serialize(obj.__dict__)
-        else:
-            return None
```

Comment:
I have added this function back except with a minor change. I have excluded the encoding part for `six.text_type` as this will be handled now by msgpack